src/napari_deeplabcut/_widgets.py

```diff
@@ -302,6 +320,40 @@ def _func():
         group.buttonClicked.connect(_func)
         return group
 
+    def _form_color_scheme_display(self, viewer):
+
+        display = ColorSchemeDisplay(parent=self)
+        # widget.remove_docked_widget(widget._color_scheme_display) where to ?
```

Comment:
Shouldn't widget removal be connected to Points.events.removed?

Reply:
I hooked this up in  on_remove